res/linux/org.mixxx.Mixxx.metainfo.xml

```diff
@@ -1907,7 +1907,7 @@
  </ul>
 </description>
     </release>
-    <release version="2.3.6" type="development" date="2023-05-29" timestamp="1685366841">
+    <release version="2.3.6" type="development" date="2023-05-19" timestamp="1684533240">
```

Comment:
The changes in this file are clearly unrelated

Reply:
Yeah, pre-commit made this change.

Reply:
This was a failed merge conflict resolve. 
In this case it is reasonable to remove this change via rebase.

Reply:
Got it. Will do this tomorrow.

Reply:
Thank you.

Reply:
Done